### PR TITLE
Restore proj-fuzzing/grizzly-reduce-worker-windows

### PR DESCRIPTION
**Pull request by petemoore**
Sorry about this @jschwartzentruber @pyoor !

This should restore your previous proj-fuzzing/grizzly-reduce-worker-windows worker pool machine image.

### CODE REVIEW

File: imagesets/generic-worker-win2016-amd/bootstrap.ps1 [hunk 1]
```diff
@@ -0,0 +1,222 @@
+$TASKCLUSTER_VERSION = "44.21.0"
```
Comment (matt-boris) [inline imagesets/generic-worker-win2016-amd/bootstrap.ps1:1]: Wanna bump to v46.1.0?
```suggestion
$TASKCLUSTER_VERSION = "v46.1.0"
```

File: imagesets/generic-worker-win2016-amd/bootstrap.ps1 [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+# install chocolatey package manager
+Invoke-Expression ($client.DownloadString('https://chocolatey.org/install.ps1'))
+
+# install nssm
+Expand-ZIPFile -File "C:\nssm-2.24.zip" -Destination "C:\" -Url "http://www.nssm.cc/release/nssm-2.24.zip"
+
+# download generic-worker
+md C:\generic-worker
+$client.DownloadFile("https://github.com/taskcluster/taskcluster/releases/download/v${TASKCLUSTER_VERSION}/generic-worker-multiuser-windows-amd64", "C:\generic-worker\generic-worker.exe")
```
Comment (matt-boris) [inline imagesets/generic-worker-win2016-amd/bootstrap.ps1:35]:
```suggestion
$client.DownloadFile("https://github.com/taskcluster/taskcluster/releases/download/${TASKCLUSTER_VERSION}/generic-worker-multiuser-windows-amd64", "C:\generic-worker\generic-worker.exe")
```

File: imagesets/generic-worker-win2016-amd/bootstrap.ps1 [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+%nssm% set "Generic Worker" AppRestartDelay 0
+%nssm% set "Generic Worker" AppStdout c:\generic-worker\generic-worker-service.log
+%nssm% set "Generic Worker" AppStderr c:\generic-worker\generic-worker-service.log
+%nssm% set "Generic Worker" AppRotateFiles 0
+"@
+Start-Process C:\generic-worker\install.bat -Wait -NoNewWindow
+
+# download worker-runner
+md C:\worker-runner
+$client.DownloadFile("https://github.com/taskcluster/taskcluster/releases/download/v${TASKCLUSTER_VERSION}/start-worker-windows-amd64", "C:\worker-runner\start-worker.exe")
```
Comment (matt-boris) [inline imagesets/generic-worker-win2016-amd/bootstrap.ps1:60]:
```suggestion
$client.DownloadFile("https://github.com/taskcluster/taskcluster/releases/download/${TASKCLUSTER_VERSION}/start-worker-windows-amd64", "C:\worker-runner\start-worker.exe")
```

File: imagesets/generic-worker-win2016-amd/bootstrap.ps1 [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+worker:
+  implementation: generic-worker
+  service: "Generic Worker"
+  configPath: c:\generic-worker\generic-worker-config.yml
+  protocolPipe: \\.\pipe\generic-worker
+cacheOverRestarts: c:\generic-worker\start-worker-cache.json
+"@
+
+# download livelog
+$client.DownloadFile("https://github.com/taskcluster/taskcluster/releases/download/v${TASKCLUSTER_VERSION}/livelog-windows-amd64", "C:\generic-worker\livelog.exe")
```
Comment (matt-boris) [inline imagesets/generic-worker-win2016-amd/bootstrap.ps1:99]:
```suggestion
$client.DownloadFile("https://github.com/taskcluster/taskcluster/releases/download/${TASKCLUSTER_VERSION}/livelog-windows-amd64", "C:\generic-worker\livelog.exe")
```

File: imagesets/generic-worker-win2016-amd/bootstrap.ps1 [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+
+# download taskcluster-proxy
+$client.DownloadFile("https://github.com/taskcluster/taskcluster/releases/download/v${TASKCLUSTER_VERSION}/taskcluster-proxy-windows-amd64", "C:\generic-worker\taskcluster-proxy.exe")
```
Comment (matt-boris) [inline imagesets/generic-worker-win2016-amd/bootstrap.ps1:102]:
```suggestion
$client.DownloadFile("https://github.com/taskcluster/taskcluster/releases/download/${TASKCLUSTER_VERSION}/taskcluster-proxy-windows-amd64", "C:\generic-worker\taskcluster-proxy.exe")
```

**Comment by jschwartzentruber** [comment]
@matt-boris I think the intention is to restore access to the existing win2016-amd AMIs, which were removed from community-tc-config unintentionally (though not deleted, we're still using them!).

If we're going to rebuild any AMIs, let's do it in win2022 since that's what everything else is running. I'm happy to help test.

**Review by matt-boris — APPROVED** [review]

**Comment by matt-boris** [comment]
Applied:

```bash
Updating WorkerPool=proj-fuzzing/grizzly-reduce-worker-windows
Updating Role=worker-pool:proj-fuzzing/grizzly-reduce-worker-windows
```

**Comment by jschwartzentruber** [comment]
Thank you!

**Pull request merged**